readme

```diff
@@ -0,0 +1,8 @@
+To make use of MySQL DB in Docker do the following steps (Windows OS instruction):
+1. Make sure you have Docker installed. For Windows, Docker Desktop is
+sufficient solution.
+2. Open cmd or Powershell and navigate to docker-compose.yml file.
+3. Run command: docker-compose up -d.
+4. You can use database. If you do not do that, you will find an error during
+starting the application from MySql connector.
```

Comment:
Please rephrase that:
```suggestion
Wedding hall application requires database to run, without it application won't even startup.
```

Please add information about required environment properties used by docker, in order to setup database correctly:
- WEDDING_HALL_DB_ADMIN_PASSWORD
- WEDDING_HALL_DB_USER
- WEDDING_HALL_DB_PASSWORD